manifests/c/Coder/Coder/2.4.0/Coder.Coder.installer.yaml

```diff
@@ -17,6 +17,8 @@ Installers:
   NestedInstallerFiles:
     - RelativeFilePath: coder.exe
       PortableCommandAlias: coder
+  AppsAndFeaturesEntries:
+    - DisplayName: Coder (portable)

```

Comment:
```suggestion
    - DisplayName: Coder
```

Reply:
For portables, we actually control what the DisplayName written to Registry (visible in Control Panel / Apps&Features is). If I don't add this distinction, and user has a side-by-side installation of portable with the nullsoft installer, the user would not get a clear distinction of the installer type in the Control Panel. See -

![image](https://github.com/microsoft/winget-pkgs/assets/88161975/7cd1a9de-e840-4f14-9a2c-0654d9a1bd85)

Adding (portable) helps user better identify the package in Control Panel.